packages/app-builder-lib/src/electron/electronMac.ts

```diff
@@ -5,7 +5,8 @@ import { filterCFBundleIdentifier } from "../appInfo"
 import { AsarIntegrity } from "../asar/integrity"
 import { MacPackager } from "../macPackager"
 import { normalizeExt } from "../platformPackager"
-import { executeAppBuilderAndWriteJson, executeAppBuilderAsJson } from "../util/appBuilder"
+import SimplePlist from "simple-plist"
```

Comment:
Can we use `plist` npm package instead? That's the one currently used in `electron/universal`

Reply:
Good suggestion. Fixed.